src/main/java/gov/nasa/cumulus/metadata/aggregator/MetadataFilesToEcho.java

```diff
@@ -113,17 +113,17 @@ public void setDatasetValues(String shortName, String version, Boolean rangeIs36
 					(Double) boundingBox.get("lonMax"),
 					(Double) boundingBox.get("lonMin"));
 		}
+
+		if (additionalAttributes != null) {
+			setAdditionalAttributes(additionalAttributes);
+		}
 	}
 
 	public void setAdditionalAttributes(JSONObject metadata){
-		if(validateJSONObjectKeyExists(metadata, "meta") && metadata.get("meta") instanceof JSONObject){
-			// contains meta
-			JSONObject meta = (JSONObject) metadata.get("meta");
-			if(validateJSONObjectKeyExists(meta, "additionalAttributes") && meta.get("additionalAttributes") instanceof JSONObject){
-				// contains additionalAttributes
-				JSONObject additionalAttributes = (JSONObject) meta.get("additionalAttributes");
-				this.additionalAttributes = additionalAttributes;
-			}
+		if(validateJSONObjectKeyExists(metadata, "additionalAttributes") && metadata.get("additionalAttributes") instanceof JSONObject){
```

Comment:
Why did we move the additional attributes block to the top layer of the Cumulus config rather than leaving it in the meta block?

Reply:
This change is to reduce the amount of data that's passed into the lambda, since in the past i got the positioning confused and when collection goes into the lambda, the task-config string would look like this
```
"additionalAttributes": "{$.meta.collection}"
```
which is passing in the whole collection JSON block, which i felt was over kill, if compared to
```
"additionalAttributes": "{$.meta.collection.meta}"
```
(from here: https://github.jpl.nasa.gov/podaac/cumulus-deploy-tf/pull/304/files#diff-daa455f763b1cce52b2778b29db372315dc84c2631b70cdbfbdc56475719ed9aR836)

The reason a whole JSON block was sent was to keep the cfg and collection structure consistent (always having `additionalAttributes` as JSON